animations/lib/src/basics/06_animated_builder.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+class AnimatedBuilderDemo extends StatefulWidget {
+  static const String routeName = '/basics/animated_builder';
+
+  _AnimatedBuilderDemoState createState() => _AnimatedBuilderDemoState();
+}
+
+class _AnimatedBuilderDemoState extends State<AnimatedBuilderDemo>
+    with SingleTickerProviderStateMixin {
+  static const Color beginColor = Colors.deepPurple;
```

Comment:
You could just have a ColorTween constant, rather than the individual colors. Might be a good thing to show.

Reply:
Actually, no, you can't. There's no const constructor for ColorTween. 🤔

Reply:
I'll add this to the list of questions for the team :)